tests/staticfiles_tests/test_storage.py

```diff
@@ -661,6 +661,14 @@ def test_aggregating_modules(self):
                 with self.subTest(module_import=module_import):
                     self.assertIn(module_import, content)
 
+    def test_data_uri_source_mapping(self):
+        relpath = self.hashed_file_path("cached/module.js")
+        self.assertEqual(relpath, "cached/module.242b85cc6818.js")
+        with storage.staticfiles_storage.open(relpath) as relfile:
+            content = relfile.read()
+            test = b"//# sourceMappingURL=data:application/json;charset=utf8;base64,eyJ"
```

Comment:
```suggestion
            test = (
                b"//# sourceMappingURL=data:application/json;charset=utf8;base64,"
                b"eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIl9zcmMv"
            ) 
```

Reply:
I don't really like implicit string joining in Python. So I have added this with a `+` between.

Reply:
> I don't really like implicit string joining in Python. So I have added this with a `+` between.

I understand, but we cannot follow contributor personal preferences, please remove "+" and update as requested.